.github/workflows/package.yaml

```diff
@@ -68,9 +77,10 @@ jobs:
             [ "$VERSION" == "master" ] && VERSION=latest
             echo IMAGE_ID=$IMAGE_ID
             echo VERSION=$VERSION
-            docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-            docker push $IMAGE_ID:$VERSION
-          else
-            echo "Skipped on PRs"
-          fi
+            echo "::set-output name=IMAGE_TAG::$(
+              echo $IMAGE_ID:$VERSION
+            )"
 
+      - name: Build Docker Image and push to registry
+        if: ${{ github.ref }} =~ ^refs\/heads\/master|^refs\/tags\/v.*
```

Comment:
Bug here on my part - since your branch name is master, it tried to run this step but failed from bad creds

Reply:
Any suggestion of how this could be improved? I used this statement of another build step so I don't know when you want to push the container to the registry?

Reply:
There's probably a way to check the repo owner + name, we should add the check for `jmbannon/ytdl-sub`

Reply:
I'll try to look into this soon